<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassEditDialog</class>
 <widget class="QDialog" name="ClassEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="classNameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="attributeTableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>481</width>
     <height>241</height>
    </rect>
   </property>
   <property name="gridStyle">
    <enum>Qt::PenStyle::SolidLine</enum>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addAttributeButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAttributeButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClassEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClassEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
